content/resources/21st-century-integrated-digital-experience-act.md

```diff
@@ -587,11 +587,11 @@ Contact OMB’s Office of the Federal CIO at [ofcio@omb.eop.gov](mailto:ofcio@om
 
 <br>
 
-<footer>
-<h3 id="footnote-label">Citations</h3>
+<div class="dg-footnote">
+<h3 id="footnote-label">Footnotes</h3>
```

Comment:
This was the only file that was not using `Footnotes` so made this update to help it be consistent with others.